expo-maps/android/src/main/java/expo/modules/maps/googleMaps/GoogleMapsView.kt

```diff
@@ -157,11 +159,13 @@ class GoogleMapsView(context: Context) : LinearLayout(context), OnMapReadyCallba
     }
   }
 
+
   fun setCameraPosition(cameraPosition: CameraPosition) {
     updateMap {
       val cameraUpdate = CameraUpdateFactory.newLatLngZoom(
-              LatLng(cameraPosition.latitude, cameraPosition.longitude),
-              cameraPosition.zoom.toFloat())
+        LatLng(cameraPosition.latitude, cameraPosition.longitude),
```

Comment:
@szdziedzic to prevent this inconsistency in the future, set continuation indent to 2 spaces in Android Studio.